noodles/pilot_job.py

```diff
@@ -88,17 +91,19 @@ def run_online_mode(args):
                 os.chdir("noodles-{0}".format(key.hex))
 
             if args.verbose:
-                print("worker: ",
-                      job.foo.__name__,
-                      job.bound_args.args,
-                      job.bound_args.kwargs,
-                      file=sys.stderr, flush=True)
+                logger.info(
+                    f"worker: {job.foo.__name__} {job.bound_args.args} {job.bound_args.kwarg}"
+                )
+                for handler in logger.handlers:
+                    handler.flush()
```

Comment:
I'm not even 100% sure if flushing is required for the `stderr` stream, but for now I'm just mimicking the previous `print` semantics as close as possible.